docs/content/latest/yugabyte-platform/secure-universes/enable-encryption-at-rest.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Enable encryption at rest
+headerTitle: Enable encryption at rest
+linkTitle: Enable encryption at rest
+description: Enable encryption at rest
+menu:
+  latest:
+    parent: secure-universes
+    identifier: enable-encryption-at-rest
+    weight: 20
+isTocNested: true
+showAsideToc: true
+---
+
+## Encryption at rest overview
+
+Data at rest within a YugabyteDB universe should be protected from unauthorized users by encrypting it. Using the Yugabyte Platform console,
+you can:
+
+- [Encryption at rest overview](#encryption-at-rest-overview)
+- [Enable encryption at rest during universe creation](#enable-encryption-at-rest-during-universe-creation)
+- [Enable encryption at rest on an existing universe](#enable-encryption-at-rest-on-an-existing-universe)
+- [Back up and restore data from encrypted at rest universe](#back-up-and-restore-data-from-encrypted-at-rest-universe)
+- [Rotate the encryption at rest keys](#rotate-the-encryption-at-rest-keys)
```

Comment:
nit on the wording, but I'd say something like "rotate universe keys used for encryption at rest" since this makes it unclear whether we mean to rotate the universe key or data keys (we do not support the latter)

Reply:
Fixed. Good idea.